RoadTripMoustache/kana_to_kanji
# Align the linear progress indicator

## 📖 Description


WIP

### ⁉️ Related Issues




Closes #25 


### 🖼️ Screenshots:


<details>
    <summary>Demo</summary> 
    
https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/5e71467a-8720-4d92-b003-ec61fb06432e

</details>

### 🧪 How to test the change?




1. Open the application
2. Start a quiz
3. See changes

### ☑️ Checklist before requesting a review
